### PR TITLE
Display

### PR DESCRIPTION
**Pull request by jwdotpark**
Install libraries in README list

- Display only works after wifi connection is established. Actually everything works after the connection..?
- Desk Time might seem skipping some second but it's just not being rendered every exact second
- we need to fix initial weird behavior! There's break message as soon as timer go
- also changing font color based on condition isn't done yet!!

### CODE REVIEW

File: Main/connect_mqtt.ino [hunk 1]
```diff
@@ -1,20 +1,20 @@
-void connectMQTT()
```
Comment (irina-baeva) [inline Main/connect_mqtt.ino:1]: why this file is changed? i think you need to uncommit everything what has no actual changes. I guess it is just formatting (

File: Main/connect_wifi.ino [hunk 2]
```diff
@@ -1,16 +1,16 @@
-void connectWiFi()
```
Comment (irina-baeva) [inline Main/connect_wifi.ino:1]: same for this file, does not seem that something changed?

File: Main/display_notificiation.ino [hunk 3]
```diff
@@ -0,0 +1,14 @@
+void getNotification() {
```
Comment (irina-baeva) [inline Main/display_notificiation.ino:1]: what about keeping the name of ino files same as function?

File: Main/connect_mqtt.ino (hunk 1 above)
Comment (jwdotpark) [reply Main/connect_mqtt.ino:1]: yeah this is bizarre, I only changed the delay interval temporarily but it says entire file was rewritten? :o lemme see what I can do

File: Main/display_notificiation.ino (hunk 3 above)
Comment (jwdotpark) [reply Main/display_notificiation.ino:1]: Oh you meant something like `getNotification.ino`?

The reason behind naming files `display_*` is I thought keeping `display` as a prefix when it comes to display related file could be useful when navigating in Arduino IDE cause it just lists files in alphabetical order. 

Maybe subfolder with straight naming scheme could be better but I'm not sure arduino IDE's recursive subfolder compilation.. It doesn't even show the tree structure

**Pull request merged**